app/Http/Controllers/Api/ProjectController.php

```diff
@@ -29,13 +34,23 @@ public function show(Request $request, Project $project){
 
     public function add_project(Request $request){
         $validator = Validator::make($request->all(), [
-            'project_address_id' => 'required|string',
+            'user_id'=> 'required|bigint',
```

Comment:
I don't think there is a "bigint" validation rule in laravel

Reply:
resolved